sdk/rust-examples/prime-numbers-generator/keep-primes/src/main.rs

```diff
@@ -0,0 +1,72 @@
+//! Prime Numbers Generator : Remove all non-prime numbers
+//!
+//! ## Context
+//!
+//! Remove all non-prime numbers from the number-set
+//!
+//! ## Authors
+//!
+//! The Veracruz Development Team.
+//!
+//! ## Copyright
+//!
+//! See the file `LICENSING.markdown` in the Veracruz root directory for licensing and
+//! copyright information.
+
+use std::fs;
+use anyhow;
+
+const FILENAME: &'static str = "/output/number-set.txt"; 
+
+fn main() -> anyhow::Result<()>{
+
+    let content = String::from_utf8(fs::read(FILENAME)?)?;
+
+    let mut num_vec: Vec<u32> = content
+        .trim()
+        .split(",")
+        .map(|x| x.parse::<u32>().unwrap())
```

Comment:
I am trying to figure out a better way of handling error, do you have something to consider?

Reply:
One thing I was told is to use `.expect("error message")` instead of `.unwrap()`. 
(I know your doubt is more advanced, just giving this small advice I got😅)

Reply:
I was thinking of this also, Thanks @AryanGodara !